disnake/interactions/base.py

```diff
@@ -19,6 +19,8 @@
     overload,
 )
 
+from disnake.webhook.interaction import InteractionFollowupWebhook
```

Comment:
this can be a relative import

Reply:
886028058deb5f9ec2704b028b54298e975f0be6